posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,276 @@
+[source,xml]
+----
+   <featureManager>
+        <feature>expressionLanguage-5.0</feature>
+    </featureManager>
+----
+
+More information about Expression Language 5.0 can be found in: 
+
+* link:https://jakarta.ee/specifications/expression-language/5.0/jakarta-expression-language-spec-5.0.html#changes-between-5-0-and-4-0[Changes between the Expression Language 4.0 and Expression Language 5.0]
+
+* link:https://jakarta.ee/specifications/expression-language/5.0/[Expression Language 5.0 specification]
+
+* link:{url-prefix}/docs/latest/reference/feature/expressionLanguage.html[Open Liberty Expression Language feature documentation]
+
+[#microprofile]
+== MicroProfile OpenAPI 3.1 
+
+OpenAPI is a standardised way of documenting REST APIs in a JSON or YAML format. It helps you generate and serve OpenAPI documentation for your REST applications that are built by using JAX-RS or Jakarta restfulWS. This can be useful for developers to test out the API during development, or for people using the API in production.
+
+To use MicroProfile OpenAPI 3.1, add the `mpOpenAPI-3.1` feature to your server.xml:
+[source,xml]
+----
+   <featureManager>
+        <feature>mpOpenAPI-3.1</feature>
+    </featureManager>
+----
+
+ Next, deploy your rest application and then visit the `/openapi` endpoint to view the OpenAPI documentation, or visit `/openapi/ui` for a user interface where you can browse through it. You can enhance the documentation by using annotations to add things like textual descriptions to each method. The annotations are available as a maven dependency:
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.openapi</groupId>
+    <artifactId>microprofile-openapi-api</artifactId>
+    <version>3.1-RC2</version>
+</dependency>
+----
+
+Version 3.1 is a minor release that includes the following updates:
+
+* Some Jakarta Bean Validation annotations will now be read and their restrictions will be added to object schemas.
+* Most annotations now include an `extension` attribute.
+* `@SecurityRequirementsSet` can now be used to document operations where authentication is optional or where more than one authentication mechanism is required.
+
+For more information about MicroProfile OpenAPI, see the following links:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Open Liberty MicroProfile OpenAPI guide]
+* link:{url-prefix}/docs/latest/documentation-openapi.html[Open Liberty OpenAPI documentation]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-3.1-RC2/apidocs/[MicroProfile OpenAPI 3.1 Javadoc]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-3.1-RC2/microprofile-openapi-spec-3.1-RC2.html[MicroProfile OpenAPI 3.1 specification]
+
+[#password]
+== Password Utilities 1.1 
+
+  The new `passwordUtilities-1.1` feature no longer starts the `federatedRegistry-1.0` feature or the Jakarta Connectors feature. 
```

Comment:
```suggestion
The new `passwordUtilities-1.1` feature no longer starts the `federatedRegistry-1.0` feature or the Jakarta Connectors feature. 
```

Reply:
The extra space appears to do weird things in the markdown at least in the preview.